gen/src/ser.rs

```diff
@@ -139,9 +139,9 @@ where
                     ser.serialize(serializer)
                 }
                 Special::Error(error) => Err(S::Error::custom(error)),
-                otherwise => todo!(
-                    "uh oh! Looks like we need to complete this pattern for {}!",
-                    otherwise
+                _ => Err(
```

Comment:
This no longer has the content information. Are we sure that's OK to remove?

Reply:
That's a good point! I have re-added it, thanks!